src/components/ec-input-field/ec-input-field.vue

```diff
@@ -192,6 +190,22 @@ const inputClasses = computed(() => {
   return classes;
 });
 
+function getInputTooltipOptions(): TooltipOptions {
```

Comment:
I think this should be computed

Reply:
it looses reactivity if I send a computed to vTootlitp. Not sure of the reason, but the function seems to work

Reply:
https://github.com/Ebury/chameleon/blob/2de00b219d1ad3d015917418d2847f47a1e9ba35/src/components/ec-currency-input/ec-currency-input.vue#L213 we do can use computed with tooltip